src/main/resources/application-bdc.properties

```diff
@@ -6,5 +6,8 @@ spring.datasource.username=${DATASOURCE_USERNAME}
 server.port=80
 
 dashboard.enable.extra_details=true
+dashboard.enable.bdc_hack=true
+dashboard.layout.type=bdc
```

Comment:
The dashboard enable bdc_hack was enabled in application.properties.  Should be enabled here.

Added the dashboard layout prop